lib/api/request_data.dart

```diff
@@ -0,0 +1,16 @@
+class RequestData {
+  String _method;
+  String _url;
+  var _body;
```

Comment:
```suggestion
  dynamic _body;
```
Falls body mehr als ein typ sein kann.